docs/data/joy/components/input/input.md

```diff
@@ -166,6 +166,24 @@ You can use these to customize the component with both the `sx` prop and the the
 
 {{"demo": "InputVariables.js", "hideToolbar": true}}
 
+## Third-party library integration
+
+To integrate the Input component with 3rd party libraries, create a wrapper component to map props with 3rd party component APIs.
+
+Then, use the wrapper component as a value to the `slotProps.input.component` property of the Joy UI Input.
```

Comment:
I had refer back to the original issue and read the Text Field doc linked there before I understood this. I think we need more detail. Here's how I'd write this:

```suggestion
The Input component can be integrated with third-party formatting libraries for more complex use cases. 

Create a wrapper component to map props with third-party component APIs.
Then pass that wrapper as a value to the `slotProps.input.component` property of the Joy UI Input.

The demos below illustrate how to do this with two popular libraries.
```

Reply:
Also, it's not clear to me what's meant by "map props with third-party component APIs." Is there another way to say this?

Reply:
For example, Joy Input passes `ref` (refer to the HTML input) prop to the slot but the 3rd-party expect `inputRef`, so a wrapper component (the middle man) is needed to map the props:

Joy Input `ref` prop -> Wrapper (convert `ref` to `inputRef`) -> 3rd-party Input

Reply:
How about "Create an adapter component to get the props from the Input component and map them to the third-party component APIs."